spec/app/services/market_history_dedupe_service_spec.rb

```diff
@@ -16,16 +16,26 @@
       expect(subject.dedupe(data, 'west')).to eq(nil)
     end
 
+    it 'returns nil if LocationId is 0' do
+      data = { 'LocationId' => 0 }
+      expect(subject.dedupe(data, 'west')).to eq(nil)
+    end
+
+    it 'returns nil if LocationId is not a numeric' do
+      data = { 'LocationId' => '3005' }
+      expect(subject.dedupe(data, 'west')).to eq(nil)
+    end
+
     it 'returns a StandardError if the AlbionID is not found in redis' do
-      data = { 'foo' => 'bar', 'AlbionId' => 1234 }
+      data = { 'foo' => 'bar', 'AlbionId' => 1234, 'LocationId' => 3005 }
```

Comment:
Had to make this change in the test so the new conditional wouldn't make the test fail.